.github/workflows/ci.yaml

```diff
@@ -1055,7 +1055,7 @@ jobs:
 
   integration-test-for-jdbc-sqlserver-2017:
     name: SQL Server 2017 integration test (${{ matrix.mode.label }})
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Don't we need the same change for `integration-test-for-jdbc-sqlserver-2019` and `integration-test-for-jdbc-sqlserver-2022`?

Reply:
Other SQL Server versions do not seem affected by this bug so I kept them as is.